contracts/crowdfund/src/events.rs

```diff
@@ -0,0 +1,12 @@
+use soroban_sdk::{vec, Env, Symbol};
+
+pub(crate) fn pledged_changed(e: &Env, amount: i128) {
```

Comment:
This would make it match the event topic. Also is amount the change? or the new total?
```suggestion
pub(crate) fn pledged_amount_changed(e: &Env, amount: i128) {
```

Reply:
it's the new total, I updated naming, thanks, https://github.com/stellar/soroban-example-dapp/pull/105/commits/9f05cd6beeba50a5cb7c39a150df4612877ccc41